lib/module.js

```diff
@@ -34,8 +35,21 @@ function httpModule (_moduleOptions) {
   // Default prefix
   const prefix = process.env.API_PREFIX || moduleOptions.prefix || '/'
 
+  // Headers
+  const headers = {
+    common: {
+      Accept: 'application/json, text/plain, */*'
```

Comment:
It is automatically set for Accept header. Other defaults was due to a SSR shared state bug in axios